### PR TITLE
CLDR-17893 Clarify rg validation for unit preferences

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-17893

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/ldml/tr35-info.md [hunk 1]
```diff
@@ -1239,6 +1239,13 @@ If any key-values are invalid, then they are ignored. Thus the following constru
 | -AA | invalid region 'AA'|
 
 ‡ Only the region portion is currently used, so in -rg-usabcdef the "abcdef" is ignored, whether or not it is valid.
+The table below shows when the region portion is valid or not.
+
+| Subdivision | Region | Used in -rg? | Comment |
+| --- | --- | --- | --- |
+| usca | US | Yes | Both the subdivision code (US Texas) and its region portion (US) are valid |
```
Comment (srl295) [inline docs/ldml/tr35-info.md:1246]:
```suggestion
| ustx | US | Yes | Both the subdivision code (US Texas) and its region portion (US) are valid |
```

unless I missed some news :)   - could also change Texas to California of course.

File: docs/ldml/tr35-info.md [hunk 2]
```diff
@@ -1253,9 +1260,10 @@ If there is no valid -mu value, the following steps are used to determine a regi
 
 1. If there is a valid -ms value then let USM  be the corresponding value in column 2 of the table below.
 Otherwise FR is not used. In either case continue with step 2.
-2. If there is a valid -rg region, let R be that region, and go to Compute the category.
-3. If there is a valid region in the locale, let R be that region, and go to Compute the category.
-4. Otherwise, compute the likely subtags for the locale.
+2. If there is a valid -rg region code as the prefix of its subdivision value, let R be that region, and go to Compute the category.
+	* See the table above for the examples `usca`, `usabc`, and `abcdef`
```
[the remark this answers is not in the document]
Comment (srl295) [reply docs/ldml/tr35-info.md:1264]: at least keep consistent

File: docs/ldml/tr35-info.md [hunk 3]
```diff
@@ -1239,6 +1239,13 @@ If any key-values are invalid, then they are ignored. Thus the following constru
 | -AA | invalid region 'AA'|
 
 ‡ Only the region portion is currently used, so in -rg-usabcdef the "abcdef" is ignored, whether or not it is valid.
```
Comment (srl295) [inline docs/ldml/tr35-info.md:1241]: this sentence could be merged with the following (or removed)

**Comment by srl295** [comment]
I added a merge commit to unblock, you can do a git pull to get the latest, or even hit the Squash button and it will get rid of the extra commit.

File: docs/ldml/tr35-info.md (hunk 1 above)
Comment (macchiati) [reply docs/ldml/tr35-info.md:1246]: fixing

File: docs/ldml/tr35-info.md [hunk 4]
```diff
@@ -1239,6 +1239,13 @@ If any key-values are invalid, then they are ignored. Thus the following constru
...
+The table below shows when the region portion is valid or not.
+
+| Subdivision | Region | Used in -rg? | Comment |
+| --- | --- | --- | --- |
+| usca | US | Yes | Both the subdivision code (US Texas) and its region portion (US) are valid |
+| usabc | US | Yes | The subdivision code is invalid, but its region portion (US) is valid |
+| abcdef | AB | No | Both the subdivision code and its region portion (~AB~) are invalid |
```
[the remark this answers is not in the document]
Comment (macchiati) [reply docs/ldml/tr35-info.md:1248]: I can invert them.

File: docs/ldml/tr35-info.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (macchiati) [reply docs/ldml/tr35-info.md:1264]: Switched to usut (Utah is one of the shortest state names)

**Review by srl295 — APPROVED** [review]

**Review by srl295 — DISMISSED** [review]

**Comment by macchiati** [comment]
I cleaned it up, so ready for review.

**Review by srl295 — APPROVED** [review]

**Pull request merged**